forms-flow-web/src/components/Form/Item/Edit.js

```diff
@@ -388,6 +397,22 @@ const Edit = React.memo(() => {
       });
   };
 
+
```

Comment:
Please remove lines -> 384 , 385 and check whether the title is getting saved on SAVE

Reply:
will fail, if removed